Exiled.API/Features/Roles/Scp173Role.cs

```diff
@@ -247,6 +258,23 @@ public GameObject Tantrum(bool failIfObserved = false, float cooldown = 0)
         /// <param name="targetPos">The Target Position.</param>
         public void Blink(Vector3 targetPos) => BlinkTimer.ServerBlink(targetPos);
 
+        /// <summary>
+        /// Snap a <see cref="Player"/> (Attack).
+        /// </summary>
+        /// <param name="player">The <see cref="Player"/>to snap.</param>
+        public void Snap(Player player)
```

Comment:
If we are going to have an Attack() method for every SCP, why are we not defining it as an abstract void in an IScpRole interface or such, that way it's called the same thing in all of them?

Reply:
Uhm good question, I didn't thonk about it, gonna do that then

Reply:
working on it